AUTHORS.rst

```diff
@@ -22,3 +22,5 @@ Contributors
 - Kenia Chang He <keniachang1997 AT gmail DOT com>
 - Raphael Cobe <rmcobe AT ncc DOT unesp DOT br>
 - Daniel Bruce <dbruc009 AT fiu DOT edu>
+- Antonio João Ferreira Francisco <antonio AT rednesp DOT br>
+- Vinicius Arcanjo <viniciusarcanjov AT gmail DOT com>
```

Comment:
```suggestion
- Vinicius Arcanjo <vindasil AT fiu DOT edu>
```

@ajoaoff could you apply this patch here using replacing with my `fiu.edu` email? Thanks. I should've stopped using my gmail in the commits messages.